src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -126,7 +127,7 @@ const styles = StyleSheet.create({
     paddingRight: 20,
     alignItems: 'center',
     borderBottomWidth: 0.5,
-    borderBottomColor: COLORS.balance_gray,
+    borderBottomColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure this color change aligns with the overall design system and color scheme of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->